src/config.ml

```diff
@@ -46,6 +46,13 @@ let github_webhook_secret toml_data =
   | Some secret ->
       secret
 
+let gitlab_webhook_secret toml_data =
+  match subkey_value toml_data "gitlab" "webhook_secret" with
+  | None ->
+      Sys.getenv_exn "GITLAB_WEBHOOK_SECRET"
+  | Some secret ->
+      secret
```

Comment:
Since we will probably need to refine this anyway, let's make it optional in the meantime. I.e., if `GITLAB_WEBHOOK_SECRET` is not defined, we could default to, say `GITHUB_WEBHOOK_SECRET`.